### PR TITLE
chrpath: add version 0.16

### PR DESCRIPTION
**Pull request by sknsean**

### CODE REVIEW

File: recipes/chrpath/chrpath_0.16.oe [hunk 1]
```diff
@@ -4,14 +4,19 @@
 ## http://linux.die.net/man/1/chrpath
 
 require ${PN}.inc
-SRC_URI = "http://alioth.debian.org/frs/download.php/file/3648/chrpath-0.14.tar.gz"
-SRC_URI += "file://standarddoc.patch"
-SRC_URI += "file://cross-endian.patch"
+SRC_URI = "https://alioth.debian.org/frs/download.php/latestfile/813/chrpath-0.16.tar.gz"
+#SRC_URI += "file://standarddoc.patch"
+#SRC_URI += "file://cross-endian.patch"
```
Comment (esben) [inline recipes/chrpath/chrpath_0.16.oe]: What is your opinion of these two patches?  Are they still relevant, or should they go?

In any case, leaving them in here, but commented out in recipe is not the way to go...

File: recipes/chrpath/chrpath_0.16.oe [hunk 2]
```diff
@@ -4,14 +4,19 @@
...
 
 MIRRORS = """
 http://alioth.debian.org/frs/download.php/file/3648/ ftp://mirrors2.kernel.org/yocto/yocto/yocto-1.2/sources/
 """
 
+#PACKAGES += "${PN}-doc"
+#FILES_${PN}-doc = "/doc"
```
Comment (esben) [inline recipes/chrpath/chrpath_0.16.oe]: Why are these two lines commented out?

Comment (sknsean) [inline recipes/chrpath/chrpath_0.16.oe]: Left over from before i added the line below

File: recipes/chrpath/chrpath_0.16.oe (hunk 1 above)
Comment (sknsean) [inline recipes/chrpath/chrpath_0.16.oe]: cross-endian is applied upstream
standarddoc is fixed below instead "EXTRA_OEMAKE_INSTALL = "docdir=${datadir}/doc/""

**Comment by sknsean** [comment]
fixed

**Pull request merged**